.github/workflows/ci.yml

```diff
@@ -82,7 +82,7 @@ jobs:
       matrix:
         go-version: [1.19.x, 1.x]
         platform: [ubuntu-latest, macos-latest]
-        module: [compose, couchbase, k3s, localstack, mysql, neo4j, postgres, pulsar, redis, redpanda, vault]
+        module: [compose, couchbase, k3s, localstack, mongodb, mysql, neo4j, postgres, pulsar, redis, redpanda, vault]
```

Comment:
Could you update https://github.com/testcontainers/testcontainers-go/pull/1447/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR103 consequently?

Reply:
done!